### PR TITLE
Not found page

### PR DESCRIPTION
**Pull request by lelekaz**
Makes a not found 404 page for nonexistent or expired business cards.

### CODE REVIEW

**Comment by sderacy** [comment]
Oh wait, is the difference between #100 and this PR that the not found page is in here? A lot of the code looks similar

**Review by sderacy — COMMENTED** [review]
Are we planning on merging both #100 and this PR?

File: server.js [hunk 1]
```diff
@@ -41,6 +41,8 @@ require("./routes/signup")(app, viewsPath);
 require("./routes/profile")(app, viewsPath);
 require("./routes/settings")(app, viewsPath);
 require("./routes/aboutus")(app, viewsPath);
+require("./routes/displaycard")(app, viewsPath);
+require("./routes/notfound")(app, viewsPath);
```
[the remark this answers is not in the document]
Comment (lelekaz) [reply server.js:45]: Yes, you are correct. I made this branch off of the displayCardPage.

**Comment by sderacy** [comment]
@lelekaz since the changes you make in #100 are encompassed in this pull request, wouldn't it make more sense to close #100 and to merge this one?

I wanna hear your thoughts since you made them but I don't see what #100 does different than #101 other than not having the 404 page which is going to be included.

**Comment by lelekaz** [comment]
> @lelekaz since the changes you make in #100 are encompassed in this pull request, wouldn't it make more sense to close #100 and to merge this one?
> 
> I wanna hear your thoughts since you made them but I don't see what #100 does different than #101 other than not having the 404 page which is going to be included.

Sounds good! I'm down with that

**Review by sderacy — APPROVED** [review]
Code looks good to me @lelekaz, merge it and delete the branch when you're done.

Nice job!

**Pull request merged**